Test/Unit/Model/PaymentTest.php

```diff
@@ -258,6 +264,52 @@ public function refundPayment_throwExceptionWhenBoltRespondWithError()
         $this->currentMock->refund($paymentMock, 100);
     }
 
+    /**
+     * @test
+     */
+    public function acceptPayment_success()
+    {
+        $this->mockApiResponse("merchant/transactions/review", '{"status": "completed", "reference": "ABCD-1234-XXXX"}');
+        $paymentMock = $this->createPaymentMock();
+        $this->orderMock->expects($this->once())->method('addStatusHistoryComment');
+
+        $this->assertTrue($this->currentMock->acceptPayment($paymentMock));
```

Comment:
Nothing wrong with the test, but it looks like return typehint is incorrect in `Payment.php` for `acceptPayment` and `denyPayment`. Could you update that while you're here?